index.html

```diff
@@ -1,79 +1,323 @@
-      content="Daphine | Portfolio, Software Developer from Uganda"
-    />
-    <meta
-      name="keywords"
-      content="Portfolio,Software Developer,HTML,CSS,Javascript, Daphine"
-    />
-    <meta name="author" content="Daphine" />
-    <meta name="viewport" content="width=device-width" />
-    <link rel="stylesheet" href="./style.css" />
-    <title>Portfolio | Daphine</title>
-  </head>
-  <body>
-    <div class="hero">
-      <header class="header bg-white">
-        <nav class="navbar flex">
-          <div class="navbar__logo">
-            <a href="#" class="logo link fz-18 fw-700 1h-20">Daphine</a>
-          </div>
-          <div class="humberger menu"></div>
-          <div class="humberger">
-            <img src="humberger.svg" alt="humberger icon" />
-          </div>
-        </nav>
-      </header>
-
-      <section class="head flex">
-        <h1 class="head__title fz-40 fw-700 1h-52">
-          I'm Daphine <span>Glad to see you!</span>
-        </h1>
-        <p class="head__dsc fz-16 fw-400 1h-24">
-          I'm a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
+   <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta
+         name="description"
+         content="Daphine | Portfolio, Software Developer from Uganda"
+         />
+      <meta
+         name="keywords"
+         content="Portfolio,Software Developer,HTML,CSS,Javascript, Daphine"
+         />
+      <meta name="author" content="Daphine" />
+      <meta name="viewport" content="width=device-width" />
+      <link rel="stylesheet" href="./style.css" />
+      <title>Portfolio | Daphine</title>
+   </head>
+   <body>
+      <div class="hero">
+         <header class="header bg-white">
+            <nav class="navbar flex">
+               <div class="navbar__logo">
+                  <a href="#" class="logo link fz-18 fw-700 1h-20">Daphine</a>
+               </div>
+               <div class="humberger menu"></div>
+               <div class="humberger">
+                  <img src="humberger.svg" alt="humberger icon" />
+               </div>
+            </nav>
+         </header>
+         <section class="head flex">
+            <h1 class="head__title fz-40 fw-700 1h-52">
+               I'm Daphine <span>Glad to see you!</span>
+            </h1>
+            <p class="head__dsc fz-16 fw-400 1h-24">
+               I'm a software developer! I can help you build a product , feature or
+               website Look through some of my work and experience! If you like what
+               you see and have a project you need coded, don’t hestiate to contact
+               me.
+            </p>
+            <a href="#" class="head__link">LET'S CONNECT</a>
+            <div class="assets__icons">
```

Comment:
- [ ] Kindly change link LET'S CONNECT to a header tag hx e,g h2,h3,h4 like the other reviewer requested you to do. Keep rocking 💪